CS342MemberViews/Sources/BrianPark/BrianPark.swift

```diff
@@ -0,0 +1,56 @@
+//
+// This source file is part of the CS342 2023 Application project
+//
+// SPDX-FileCopyrightText: 2023 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import InstructorViews
+import SwiftUI
+import MapKit
```

Comment:
SwiftLint is throwing an error here because imports should be sorted in alphabetical order based on the name of the package.

```suggestion
import InstructorViews
import MapKit
import SwiftUI
```